critical-user-flows/accountSettings.spec.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-ignore
+import { test, expect } from "@playwright/test";
+
+test("account settings", async ({ page }) => {
+  await page.goto("https://danube-web.shop/");
+
+  await page.click("#login");
```

Comment:
@modern-sapien it does not need to happen in this PR, but could you rewrite these to conform to the newer APIs? It should take a few minutes.

Reply:
"These" here means "the test specs / scripts"

Reply:
I have a project with them updated that I need to use for the docs updates. 

I'd like to actually shoot one your way outside of this PR for you to edit and pass back so I can know exactly you're going for.